bin/ruby-build

```diff
@@ -748,7 +748,18 @@ build_package_truffleruby() {
   build_package_copy
 
   cd "${PREFIX_PATH}"
-  "${PREFIX_PATH}/lib/truffle/post_install_hook.sh"
+  "${PREFIX_PATH}/lib/truffle/post_install_hook.sh" || return $?
```

Comment:
What is the purpose for the `|| return $?` ? It's the last line of the function so the last line's exit status will be the exit status for the function, no?